packages/dds/tree/src/feature-libraries/typed-schema/typedTreeSchema.ts

```diff
@@ -235,11 +235,12 @@ export class FlexFieldSchema<out TKind extends FlexFieldKind = FlexFieldKind> {
 	 * @privateRemarks
 	 * Alias for the constructor, but with extends clause for the `Types` parameter that {@link FlexFieldSchema} can not have (due to recursive type issues).
 	 */
-	public static create<TKind extends FlexFieldKind>(
+	public static create<TKind extends FlexFieldKind, const TMetadata = unknown>(
```

Comment:
This generic parameter doesn't do anything, is that correct? FlexFieldSchema is not typed over TMetadata, so there is no need to carry a generic.

Reply:
Ah, I missed this in my cleanup. Will remove.